docs/source/learners/rag.rst

```diff
@@ -1,2 +1,53 @@
 Retrieval Augmented Generation
 =======================================
+RAG (Retrieval Augmented Generation) learners combine the strengths of both retrieval models
```

Comment:
Please, explain a bit the methodology, maybe adding a single paragraph would be great!

Reply:
For this, you can use the paper that I shared with you `Phonexises` team.